packages/zoe/src/zoeService/makeInvitation.js

```diff
@@ -2,7 +2,16 @@
 import { assert, details as X } from '@agoric/assert';
 import { AmountMath, makeIssuerKit, AssetKind } from '@agoric/ertp';
 
-export const createInvitationKit = shutdownZoeVat => {
+/**
+ * @param {ShutdownWithFailure | undefined} shutdownZoeVat
+ * @param {TranslateFee | (() => void)} translateFee
+ * @param {TranslateExpiry | (() => void)} translateExpiry
+ */
+export const createInvitationKit = (
+  shutdownZoeVat = undefined,
+  translateFee = () => {},
+  translateExpiry = () => {},
```

Comment:
Since you're using the results of these functions, if you really want them to default to returning `undefined`, which is their current behavior, better to make this more explicit.

```suggestion
  translateFee = () => undefined,
  translateExpiry = () => undefined,
```

But is returning `undefined` in fact the default behavior you really want?

Reply:
Hmm, what do you think would be preferable?

Reply:
I just looked at where they're used, and we have a worse problem. That same problematic `if` says
```js
if (timeAuthority && expiry && fee) {
```
which interprets falsy as ignore. But `expiry` is absolute time according to some time authority, and `0` is falsy. Even if it could never happen with the only possible time authority, this code should not be written to assume that `0` is not an absolute time that might matter.

Given that, I would keep the `undefined` from my suggestion above (still to be made explicit). But please revise that `if` to check `=== undefined` explicitly.

Reply:
Done